### ISSUE TITLE
Instance type is ignored

### ISSUE THREAD

**Issue by alex-cortecs** [cortecs-ai/cortecs-py#6]
When using 
```
with DedicatedLLM(client=cortecs, model_name='Qwen/Qwen2-7B-Instruct', instance_type='NVIDIA_L40S_1') as llm:
    essay = llm.invoke('Write an essay about dynamic provisioning')
    print(essay.content)
```
the instance_type is ignored and the recommended instance type is used.

**Comment by alex-cortecs** [comment]
Okay now I understand. I deleted all models in my console and after that choosing instance type works.

**Comment by eva-jagodic** [comment]
Fixed with c464835b4aa69b79f1fe8a6bf85d168ac2d93722.
